src/components/Create/components/RewardsList/AddEditRewardModal.tsx

```diff
@@ -14,8 +14,9 @@ import { useWallet } from 'hooks/Wallet'
 import { UploadRequestOption } from 'rc-upload/lib/interface'
 import { useCallback, useContext, useEffect, useState } from 'react'
 import { stopPropagation } from 'react-stop-propagation'
-import { restrictedIpfsUrl } from 'utils/ipfs'
+import { openIpfsUrl } from 'utils/ipfs'
```

Comment:
note: since this url will be origin-restricted, we shouldn't have any trouble using it everywhere and just calling it `ipfsUrl`